packages/vendure-order-client/test/vendure-order-client.spec.ts

```diff
@@ -59,6 +86,66 @@ describe('Vendure order client', () => {
     expect(server.app.getHttpServer).toBeDefined;
   });
 
+  it('Create a promotion', async () => {
+    await adminClient.asSuperAdmin();
+    const promotion = await createPromotion(
+      adminClient as any,
+      couponCodeName,
+      'order_fixed_discount',
+      [
+        {
+          name: 'discount',
+          value: '10',
+        },
+      ]
+    );
+    expect(promotion.name).toBe(couponCodeName);
+    expect(promotion.couponCode).toBe(couponCodeName);
+  });
+
+  it('Creates Shipping Method', async () => {
```

Comment:
Can we try to avoid admin calls `create shippingmethod` and `create paymentmethod`? The client has its own  non-shared initialData file, so you can add paymentMethod there. Shipping method should already be in there.

This package really focusses on the clientside, so thats why I would like to avoid any unnecessary admin mutation in its e2e tests.

Let me know what you think